html/icon-fontawesome.html

```diff
@@ -167,1218 +132,21 @@
-                                    <div class="col-sm-6 col-md-4 col-lg-3"><i class="fa fa-rotate-left"></i>
-                                        fa-rotate-left <span class="text-muted">(alias)</span> </div>
-                                    <div class="col-sm-6 col-md-4 col-lg-3"><i class="fa fa-rotate-right"></i>
-                                        fa-rotate-right <span class="text-muted">(alias)</span> </div>
-                                    <div class="col-sm-6 col-md-4 col-lg-3"><i class="fa fa-save"></i> fa-save <span
-                                            class="text-muted">(alias)</span> </div>
-                                    <div class="col-sm-6 col-md-4 col-lg-3"><i class="fa fa-scissors"></i> fa-scissors
-                                    </div>
-                                    <div class="col-sm-6 col-md-4 col-lg-3"><i class="fa fa-strikethrough"></i>
-                                        fa-strikethrough</div>
-                                    <div class="col-sm-6 col-md-4 col-lg-3"><i class="fa fa-subscript"></i> fa-subscript
-                                    </div>
-                                    <div class="col-sm-6 col-md-4 col-lg-3"><i class="fa fa-superscript"></i>
-                                        fa-superscript</div>
-                                    <div class="col-sm-6 col-md-4 col-lg-3"><i class="fa fa-table"></i> fa-table</div>
-                                    <div class="col-sm-6 col-md-4 col-lg-3"><i class="fa fa-text-height"></i>
-                                        fa-text-height</div>
-                                    <div class="col-sm-6 col-md-4 col-lg-3"><i class="fa fa-text-width"></i>
-                                        fa-text-width</div>
-                                    <div class="col-sm-6 col-md-4 col-lg-3"><i class="fa fa-th"></i> fa-th</div>
-                                    <div class="col-sm-6 col-md-4 col-lg-3"><i class="fa fa-th-large"></i> fa-th-large
-                                    </div>
-                                    <div class="col-sm-6 col-md-4 col-lg-3"><i class="fa fa-th-list"></i> fa-th-list
-                                    </div>
-                                    <div class="col-sm-6 col-md-4 col-lg-3"><i class="fa fa-underline"></i> fa-underline
-                                    </div>
-                                    <div class="col-sm-6 col-md-4 col-lg-3"><i class="fa fa-undo"></i> fa-undo</div>
-                                    <div class="col-sm-6 col-md-4 col-lg-3"><i class="fa fa-unlink"></i> fa-unlink <span
-                                            class="text-muted">(alias)</span> </div>
-                                </div>
-                            </div>
-                        </div>
-                        <div class="card">
-                            <div class="card-body">

```

Comment:
Please do not remove div class="card-body" in line 1070
Just remove all data inside them so that data can represented in these empty card as and when required.

Reply:
changes done